try{
    addlert("hello");
}
catch(err){
    console.log(err);
}